### PR TITLE
lsst science pipeline update

### PR DESCRIPTION
**Pull request by ijiraq**
This PR makes deployment of the lsst science pipeline latest releases so that ARCADE keeps up.  This should become part of a CI/CD process, eventually.

### CODE REVIEW

**Comment by brianmajor** [comment]
I'll wait for the approval by @sfabbro before merging this.

**Review by sfabbro — APPROVED** [review]
Looks good to me.

**Pull request merged**
